tests/test_run_cmd_llm_infer.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import subprocess
+import sys
+import time
+from pathlib import Path
+
+import pytest
+
+sys.path.append(str(Path(__file__).absolute().parents[1]))
+
+
+def test_run_cmd_llm_infer():
+    """
+    Starts (if available) TRTLLM, Nemo, and VLLM servers, then sends the same prompt
+    with top_logprobs=1. It then compares the logprobs for each token across the models.
+    """
+    model_info = [
+        ("vllm", 'Qwen/Qwen2.5-0.5B-Instruct'),
+    ]
+
+    outputs_map = {}
+    for server_type, model_path in model_info:
+        if not model_path:
+            continue
+
+        output_dir = f"/tmp/nemo-skills-tests/qwen2.5/{server_type}-run-cmd"
+        command = (
+            f"cd /nemo_run/code/tests/scripts/ && "
+            f"rm -f {os.path.join(output_dir, 'output.txt')} && "
```

Comment:
it's better to remove this file outside of the command. Otherwise if the run_cmd fails completely the assert can be passed if the file is present in the test folder. You can use the newly added docker_rm utility for this to avoid permission errors